vdr/store/store.go

```diff
@@ -47,20 +47,26 @@ const (
 type store struct {
 	db            stoabs.KVStore
 	storeProvider storage.Provider
+	// inMemory is a temporary change to let the current store be in-memory for test.
+	inMemory bool
 }
 
 func (s *store) Name() string {
 	return "DID Document Store"
 }
 
 // NewStore returns an instance of a VDR store
-func NewStore(storeProvider storage.Provider) vdr.Store {
-	return &store{storeProvider: storeProvider}
+func NewStore(storeProvider storage.Provider, inMemory bool) vdr.Store {
+	return &store{storeProvider: storeProvider, inMemory: inMemory}
 }
 
 func (s *store) Configure(_ core.ServerConfig) error {
 	var err error
-	s.db, err = s.storeProvider.GetKVStore(didStoreName, storage.PersistentStorageClass)
+	if s.inMemory {
+		s.db, err = s.storeProvider.GetKVStore(didStoreName, storage.VolatileStorageClass)
```

Comment:
Volatile doesn't mean in-memory per se. Just that the storage may be lost without losing data (e.g. connection backoffs).

Other tests use `StaticKVProvider` with a precreated test-BBolt store (`CreateTestBBoltStore`, `storage/test.go:38`).